sdv/metadata/table.py

```diff
@@ -584,6 +584,23 @@ def filter_valid(self, data):
 
         return data
 
+    def make_ids_unique(self, data):
+        """Repopulate any id fields in provided data to guarantee uniqueness.
+
+        Args:
+            data (pandas.DataFrame):
+                Table data.
+
+        Returns:
+            pandas.DataFrame:
+                Table where all id fields are unique.
+        """
+        for name, field_metadata in self._fields_metadata.items():
+            if field_metadata['type'] == 'id':
+                data[name] = self._make_ids(field_metadata, len(data))
```

Comment:
This returns all the ids in order. Idk if we want to randomize them, or provide the option for that

Reply:
There is an issue about this #329 , we will solve that later.